iml-gui/crate/Cargo.toml

```diff
@@ -28,6 +29,7 @@ js-sys = "0.3"
 lazy_static = "1.4.0"
 natord = "1.0"
 number-formatter = { path = "../../number-formatter", version = "0.1" }
+rand_core = "0.5.1"
```

Comment:
Why is this not a dev-dependency?

Reply:
Forgot to remove after `shuffle` removal.